entries/u/unstoppabledomains.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Unstoppable Domains": {
+    "domain": "unstoppabledomains.com",
+    "img": "unstoppabledomains.com.png",
```

Comment:
Please use this SVG version of the logo instead:
![](https://raw.githubusercontent.com/ApCoder123/ApCoder123/master/5779.svg)